src/content/docs/guides/declare-module.md

```diff
@@ -48,7 +48,7 @@ It is highly recommended to use `.ts`, `.js` files for your translations, so you
 :::
 
 :::warning
-If you use json for this (not recommended), you must add `../` to the path in your `seyfert.config.js` because typescript ignores jsons that are not imported manually.
+If you use json for this (not recommended), you must add `../` to the path in your `seyfert.config.js` as typescript ignores json files that are not imported manually.
```

Comment:
are u sure "as" is correct here instead of "because"??

Reply:
yes as both values are possible but "as" in this sentence is more clearly and direct.